witness/golang/omniwitness/usbarmory/usbnet.go

```diff
@@ -0,0 +1,133 @@
+// https://github.com/f-secure-foundry/tamago-example
+//
+// Copyright (c) F-Secure Corporation
+// https://foundry.f-secure.com
+//
+// Use of this source code is governed by the license
+// that can be found in the LICENSE file.
+
+//go:build usbarmory
+// +build usbarmory
+
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net"
+	"net/http"
+	"strings"
+	"time"
+
+	usbnet "github.com/f-secure-foundry/imx-usbnet"
+	"github.com/f-secure-foundry/tamago/soc/imx6/usb"
+	"github.com/golang/glog"
+	"github.com/miekg/dns"
+)
+
+const (
+	deviceIP  = "10.0.0.1"
+	deviceMAC = "1a:55:89:a2:69:41"
+	hostMAC   = "1a:55:89:a2:69:42"
+	resolver  = "8.8.8.8:53"
```

Comment:
Might be worth popping in a TODO to think about DHCP?

Reply:
I'm reasonably sure we will need to do this at some point, but happy to leave the TODO out - as you say, it'll be obvious when we need it.